python/src/opendp/polars.py

```diff
@@ -11,6 +20,28 @@
 
 
 class DPExpr(object):
+    '''
+    If both ``opendp`` and ``polars`` have been imported,
+    the methods of :py:class:`DPExpr` are registered under the ``dp`` namespace in
+    `Polars expressions <https://docs.pola.rs/py-polars/html/reference/expressions/index.html>`_.
+    An expression can be used as a plan in :py:func:`opendp.measurements.make_private_lazyframe`;
+    See the full example there for more information.
+
+    This class is typically not used directly by users:
+    Instead its methods are registered under the ``dp`` namespace of Polars expressions.
+    
+    However, it can be useful if stronger typing is desired:
+    Both of these expressions return a ``DPExpr``:
+
+    >>> import polars as pl
+    >>> pl.len().dp
+    <opendp.polars.DPExpr object at ...>
+    >>> DPExpr(pl.len())
+    <opendp.polars.DPExpr object at ...>
+
+    However, a static analysis tool like `mypy <https://mypy.readthedocs.io/en/stable/>`_
+    will see the type of the first expression as ``Any``, while the later is ``DPExpr``.
```

Comment:
I think you'd like us to say something like this? I feel like it might be introducing unnecessary complexity: Libraries that aren't very strictly typed are not a surprise, and if the user wanted stricter typing they could do this:

```python
dp_expr: dp.DPExpr = pl.len().dp
```

Reply:
I'm fine with it not being mentioned if you prefer! Your judgement. If you are familiar with mypy typing then this may be apparent. The most important thing to communicate in this docstring is that these methods are accessible under .dp, which you've already handled.